### ISSUE TITLE
mat-chip listboxes are missing labels in examples

### ISSUE THREAD

**Issue by mfairchild365** [angular/material2#15010]
#### What is the expected behavior?
The purpose of a `listbox` that contains the chips must be conveyed to screen reader users.

#### What is the current behavior?
The purpose of listboxes in the `mat-chip` examples are not conveyed to screen reader users.

#### What are the steps to reproduce?
1. Using either of the following screen readers
   - NVDA 2018.4.1 + FireFox 64
   - VoiceOver for IOS (12.1.3) + Safari
2. Open the [chips example](https://material.angular.io/components/chips/examples)
3. Navigate to the first chip in the 'basic chips' example
   - In NVDA you can press the tab key until you reach the chip
   - In VoiceOver for iOS you can swipe right until you reach the chip
4. Observe that a label was not announced for the group of listbox options.

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
- Angular Material Example 7.2.2 located at https://material.angular.io/components/
- NVDA 2018.4.1 + FireFox 64
- VoiceOver for IOS (12.1.3) + Safari (12.1.3)

#### Is there anything else we should know?
The documentation for chips states the following:

> The chip input should have a placeholder or be given a meaningful label via `aria-label` or `aria-labelledby`

The [ARIA spec for the `listbox` role](https://www.w3.org/TR/wai-aria-1.1/#listbox) states that the author is required to provide an accessible name, which can be achieved via `aria-label` or `aria-labelledby`.
